types/constants.go

```diff
@@ -13,42 +13,70 @@ import (
 )
 
 var (
-	BlockFrequency         BlockHeight
-	BlockSizeLimit         = uint64(2e6)
+	// BlockFrequency {help}
+	BlockFrequency BlockHeight
+	// BlockSizeLimit {help}
+	BlockSizeLimit = uint64(2e6)
+	// ExtremeFutureThreshold {help}
 	ExtremeFutureThreshold Timestamp
-	FutureThreshold        Timestamp
-	GenesisBlock           Block
+	// FutureThreshold {help}
+	FutureThreshold Timestamp
+	// GenesisBlock is the first block of the block chain
+	GenesisBlock Block
 
-	// The GenesisID is used in many places. Calculating it once saves lots of
+	// GenesisID is used in many places. Calculating it once saves lots of
```

Comment:
articles like `The` should be allowed by golint

Reply:
golint follows a certain format for comments and so it fails with this message when linting:

types/constants.go:38:2: comment on exported var GenesisID should be of the form "GenesisID ..."

Reply:
apparently, constants are not allowed to have leading articles: https://github.com/golang/lint/issues/11. That's why it worked on some comments, for example: in block.go. BlockHeader was defined as type whereas in constants.go, GenesisBlock was defined as var.

Reply:
huh. That's strange, but good to know.